lib/matplotlib/backends/_backend_tk.py

```diff
@@ -894,6 +895,24 @@ def set_history_buttons(self):
         if "Forward" in self._buttons:
             self._buttons['Forward']['state'] = state_map[can_forward]
 
+class NavigationToolbar2Ttk(NavigationToolbar2Tk):
+    def _Button(self, text, image_file, toggle, command):
+        im = tk.PhotoImage(master=self, file=image_file)
+        b = ttk.Button(self, text=text, padding=(2, 2), image=im, command=command)
+        b._ntimage = im
+        b.pack(side='left')
+        return b
```

Comment:
This implementation appears outdated compared to the original `_Button`.

Reply:
I'm not sure I see what you mean; the original `_Button` uses the standard `tk.Button`, which has a rather different implementation and API than `ttk.Button`, which is used here.